uber/site_sections/summary.py

```diff
@@ -225,13 +225,13 @@ def restricted_untaken(self, session):
                     untaken[job.location][hour].append(job)
         flagged = []
         for attendee in attendees:
-            if attendee.trusted and not attendee.is_dept_head:
+            if not attendee.is_dept_head:
                 overlapping = defaultdict(set)
                 for shift in attendee.shifts:
                     if not shift.job.restricted:
                         for dept in attendee.assigned_depts_ints:
                             for hour in shift.job.hours:
-                                if hour in untaken[dept]:
+                                if attendee.trusted_in(dept) and hour in untaken[dept]:
                                     overlapping[shift.job].update(untaken[dept][hour])
```

Comment:
At a glance, this looks correct.

The purpose of this page is a report for Stops of "which trusted staffers signed up for regular shifts instead of restricted ones, even though the regular shifts were literally at the same time as the restricted ones?"  This is a useful question because we often have people just sign up for a shift without looking too carefully, and we often have more trouble filling the restricted shifts than the regular ones.

Your change looks good at a glance.  Shame we don't have enough unit tests to have as much confidence in this kind of change :(